src/Context/FixtureContext.php

```diff
@@ -687,6 +695,42 @@ class_exists($extension) && is_subclass_of($extension, Extension::class),
         }
     }
 
+    /**
+     * Get the module that represents the application.
+     */
+    protected function getAppModule(): Module
+    {
+        $project = ModuleManifest::config()->get('project') ?: 'mysite';
```

Comment:
```suggestion
        $project = ModuleManifest::config()->get('project') ?: 'app';
```
More likely to be `app` these days?

Reply:
This is just copied from `getDestinationConfigFolder()`, which I've refactored it out from. I can change it if you want, but that's the context.